test-integration/filter-aaveV1.js

```diff
@@ -1,147 +1,64 @@
-    const weth = await WETH.new();
-    uniswapRouter = await UniswapV2Router01.new(uniswapFactory.address, weth.address);
-
-    // deploy Argent
-    registry = await Registry.new();
-    tokenRegistry = await TokenRegistry.new();
-    dappRegistry = await DappRegistry.new(0);
-    guardianStorage = await GuardianStorage.new();
-    transferStorage = await TransferStorage.new();
-    module = await ArgentModule.new(
-      registry.address,
-      guardianStorage.address,
-      transferStorage.address,
-      dappRegistry.address,
-      uniswapRouter.address,
-      SECURITY_PERIOD,
-      SECURITY_WINDOW,
-      RECOVERY_PERIOD,
-      LOCK_PERIOD);
-    await registry.registerModule(module.address, ethers.utils.formatBytes32String("ArgentModule"));
-    filter = await AaveV1LendingPoolFilter.new();
-    aTokenFilter = await AaveV1ATokenFilter.new();
+    const filter = await AaveV1LendingPoolFilter.new();
+    const aTokenFilter = await AaveV1ATokenFilter.new();
     const approveFilter = await OnlyApproveFilter.new();
-    await dappRegistry.addDapp(0, aaveLendingPoolCore, approveFilter.address);
-    await dappRegistry.addDapp(0, aaveLendingPool.address, filter.address);
-    await dappRegistry.addDapp(0, aToken.address, aTokenFilter.address);
-    await dappRegistry.addDapp(0, aUSDCToken.address, aTokenFilter.address);
-    await dappRegistry.addDapp(0, relayer, ZERO_ADDRESS);
-    walletImplementation = await BaseWallet.new();
-    manager = new RelayManager(guardianStorage.address, tokenRegistry.address);
+    await argent.dappRegistry.addDapp(0, aaveLendingPoolCore, approveFilter.address);
+    await argent.dappRegistry.addDapp(0, aaveLendingPool.address, filter.address);
+    await argent.dappRegistry.addDapp(0, aToken.address, aTokenFilter.address);
+    await argent.dappRegistry.addDapp(0, aUSDCToken.address, aTokenFilter.address);
   });
 
   beforeEach(async () => {
-    // create wallet
-    const proxy = await Proxy.new(walletImplementation.address);
-    wallet = await BaseWallet.at(proxy.address);
-    await wallet.init(owner, [module.address]);
-
-    // fund wallet
-    await wallet.send(10000000);
-
-    await initNonce(wallet, module, manager, SECURITY_PERIOD);
+    wallet = await argent.createFundedWallet({ USDC: "100" });
   });
 
   describe("deposit", () => {
     it("should allow deposits of ETH on behalf of wallet", async () => {
-      const transactions = encodeCalls([
+      const transactions = utils.encodeCalls([
         [aaveLendingPool, "deposit", [AAVE_ETH_TOKEN, 1000, ""], 1000]
       ]);
 
-      const txReceipt = await manager.relay(
-        module,
-        "multiCall",
-        [wallet.address, transactions],
-        wallet,
-        [owner],
-        1,
-        ETH_TOKEN,
-        relayer);
-
-      const { success, error } = utils.parseRelayReceipt(txReceipt);
+      const receipt = await argent.multiCallRaw(wallet, transactions);
```

Comment:
we could use `argent.multicall` instead of `argent.multicallRaw` wherever possible to reduce verbosity

Reply:
Done